servers/rendering_server.h

```diff
@@ -1385,9 +1385,11 @@ class RenderingServer : public Object {
 	virtual void canvas_item_set_visibility_notifier(RID p_item, bool p_enable, const Rect2 &p_area, const Callable &p_enter_callbable, const Callable &p_exit_callable) = 0;
 
 	enum CanvasGroupMode {
+		// Note: The following need to match the order of CanvasItem::ClipChildrenMode
 		CANVAS_GROUP_MODE_DISABLED,
 		CANVAS_GROUP_MODE_CLIP_ONLY,
 		CANVAS_GROUP_MODE_CLIP_AND_DRAW,
+		CANVAS_GROUP_MODE_SUBTRACT,
```

Comment:
New enum fields must be added at the end to avoid breaking compatibility. If we need `CanvasItem::ClipChildrenMode` to share the same values[^1] we can be explicit about the values:

```C++
enum ClipChildrenMode {
	CLIP_CHILDREN_DISABLED = RS::CANVAS_GROUP_MODE_DISABLED, // 0
	CLIP_CHILDREN_ONLY = RS::CANVAS_GROUP_MODE_CLIP_ONLY, // 1
	CLIP_CHILDREN_AND_DRAW = RS::CANVAS_GROUP_MODE_CLIP_AND_DRAW, // 2
	CLIP_CHILDREN_SUBTRACT = RS::CANVAS_GROUP_MODE_SUBTRACT, // 4
	CLIP_CHILDREN_MAX = 5,
};
```

[^1]: I'm assuming because of https://github.com/godotengine/godot/blob/3ed4497113fa10611b90290ce22a751fb9d26e2e/scene/main/canvas_item.cpp#L1454

Reply:
Even if we're explicit about the values, it would break here:
https://github.com/godotengine/godot/blob/3ed4497113fa10611b90290ce22a751fb9d26e2e/scene/main/canvas_item.cpp#L1203
There's no way to be explicit about _these_ values. And a patch to enable such option would probably be rather large to be included into this PR.

Reply:
`PROPERTY_HINT_ENUM` allows you to specify the values explicitly (See https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_exports.html#exporting-enums). So this should work:

```C++
ADD_PROPERTY(PropertyInfo(Variant::INT, "clip_children", PROPERTY_HINT_ENUM, "Disabled:0,Clip Only:1,Clip + Draw:2,Subtract:4"), "set_clip_children_mode", "get_clip_children_mode"); 
```

For example, we do this in the `Control::anchors_preset` property:

https://github.com/godotengine/godot/blob/5f6524ad445a1ac072429e4e3c42eab84c0a869d/scene/gui/control.cpp#L3493-L3498

Reply:
It worked, thanks